pkg/bot/mattermost.go

```diff
@@ -163,7 +164,10 @@ func (mm *mattermostMessage) handleMessage(b MMBot) {
 	mm.log.Debugf("Received mattermost event: %+v", mm.Event.Data)
 
 	// Trim the @BotKube prefix if exists
-	mm.Request = strings.TrimPrefix(post.Message, "@"+b.BotName+" ")
+	mm.Request = strings.TrimPrefix(post.Message, fmt.Sprintf("@%s ", b.BotName))
+
+	// Trim the @botkube prefix if still exists in request
+	mm.Request = strings.TrimPrefix(mm.Request, fmt.Sprintf("@%s ", strings.ToLower(b.BotName)))
```

Comment:
If you agree with totally ignoring the casing, here we should trim prefix regardless of the casing (I think the easiest way would be to use regex with the `i` flag)